plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/Plugin.java

```diff
@@ -0,0 +1,21 @@
+package org.jenkins.tools.test.model;
+
+import hudson.util.VersionNumber;
+
+public class Plugin {
```

Comment:
I think you should really rename this to `PCTPlugin` or so to avoid headaches like below with many classes named `Plugin`.

Reply:
Good idea. Wasn't a fan of the name either.